website/components/ComingSoon.tsx

```diff
@@ -26,11 +25,12 @@ function ComingSoon() {
       <Subtitle as="p" className={subtitleStyle}>
         Coming soon!
       </Subtitle>
-      <Image
-        src="/images/coming-soon.gif"
+      <img
+        src={`${process.env.NEXT_PUBLIC_CDN}/images/guidelines/coming-soon.gif`}
```

Comment:
where does `NEXT_PUBLIC_CDN` come from? does it need to be added to the CSP?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src#violation_cases

Reply:
🤦 I should document this somewhere. I added our CDN URL to `.env.local`
Is there a better place to put this?

Reply:
`process.env` is not available in the browser right? would this be a problem if React on the client needs to hydrate or re-render the component?

Reply:
Next makes it available in the browser: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
Is there any benefit to making it an environment variable versus a constant exported from a file?

Reply:
Just that we don't have to import it, but I suppose the cost is higher than the benefit here. So i can just add the cdn path to our utils folder

Reply:
do you know what's the state of the Content Security Policy?